jdaviz/configs/cubeviz/plugins/tests/test_tools.py

```diff
@@ -47,6 +52,9 @@ def test_spectrum_at_spaxel(cubeviz_helper, spectrum1d_cube_with_uncerts):
         {'event': 'mouseleave', 'domain': {'x': x, 'y': y}, 'altKey': False})
     assert flux_viewer.toolbar.active_tool._mark.visible is False
 
+    # Check that X is still zoomed but Y is reset.
+    assert_allclose(spectrum_viewer.get_limits(), (4.623e-07, 4.6232e-07, 28, 92))
```

Comment:
On `main` without this patch, this test would fail as such:

```
        # Check that X is still zoomed but Y is reset.
>       assert_allclose(spectrum_viewer.get_limits(), (4.623e-07, 4.6232e-07, 28, 92))

jdaviz/configs/cubeviz/plugins/tests/test_tools.py:56:

E           AssertionError:
E           Not equal to tolerance rtol=1e-07, atol=0
E
E           Mismatched elements: 2 / 4 (50%)
E           Max absolute difference among violations: 4.00276968e-11
E           Max relative difference among violations: 8.65800676e-05
E            ACTUAL: array([4.6228e-07, 4.6236e-07, 2.8000e+01, 9.2000e+01])
E            DESIRED: array([4.6230e-07, 4.6232e-07, 2.8000e+01, 9.2000e+01])
```

Reply:
I updated the test to also account for Y limits, but you get the idea.